test/slider-test.jsx

```diff
@@ -93,6 +93,20 @@ describeWithDOM('<Slider />', () => {
       assert.isTrue(pitRender.calledOnce, 'one pit was rendered only once');
     });
 
+    it('should set focus on handle if autoFocus prop is true', () => {
+      const wrapper = mount(<Slider autoFocus />);
+      const focusedElement = document.activeElement.outerHTML;
+      const elem = wrapper.find('button').html();
+      assert.equal(elem, focusedElement, 'Slider\'s handle is focused');
+    });
+
+    it('should set focus on first handle when multiple handles are available', () => {
+      const wrapper = mount(<Slider values={[0, 50, 100]} autoFocus />);
+      const focusedElement = document.activeElement.outerHTML;
```

Comment:
rather than using outerHTML in these tests, could we put an attribute on the element, and compare that?

Reply:
JSDom doesn't expose custom attributes (Last time I checked). It does forward the ID, but it wouldn't work on the multiple handles test...

Reply:
even data attributes?

Reply:
Wrote a custom handle that will generate random IDs to use on the tests (instead of comparing the full generated HTMLs).